columnq/src/columnq.rs

```diff
@@ -248,13 +262,34 @@ mod tests {
 
     #[test]
     fn azure_object_store_type() {
-        let unknown = "az://bucket_name/path";
+        let host_url = "az://bucket_name/path";
+        let provider = ColumnQObjectStoreProvider {};
+        // https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio#http-connection-strings
+        env::set_var("AZURE_STORAGE_ACCOUNT_NAME", "devstoreaccount1");
+        env::set_var("AZURE_STORAGE_ACCOUNT_KEY", "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==");
+
+        let res = provider
+            .get_by_url(&Url::from_str(host_url).unwrap());
+        let msg = match res {
+            Err(e) => format!("{}", e),
+            Ok(_) => "".to_string(),
+        };
+        assert_eq!("".to_string(), msg);
+
+        env::remove_var("AZURE_STORAGE_ACCOUNT_NAME");
+        env::remove_var("AZURE_STORAGE_ACCOUNT_KEY");
+    }
+
+    #[test]
+    fn unknown_object_store_type() {
+        let unknown = "unknown://bucket_name/path";
         let provider = ColumnQObjectStoreProvider {};
         let err = provider
             .get_by_url(&Url::from_str(unknown).unwrap())
             .unwrap_err();
+        println!("{:?}", err.to_string());
         assert!(err
             .to_string()
-            .contains("Unsupported object store scheme az"))
+            .contains("External error: Invalid table URI: Unsupported scheme: \"unknown\""))
```

Comment:
[for reviewer] error message changes because it is now from `BlobStoreType::try_from(url_schema)`